homeassistant/components/switchbot/config_flow.py

```diff
@@ -0,0 +1,92 @@
+"""Config flow for Switchbot."""
+import logging
+
+import pygatt
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow
+from homeassistant.const import CONF_MAC, CONF_NAME, CONF_PASSWORD, CONF_SENSOR_TYPE
+
+from .const import ATTR_BOT, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def _btle_connect(mac):
+    """Try to connect to switchbot device."""
+    adapter = pygatt.GATTToolBackend()
+
+    try:
+        adapter.start()
+        device = adapter.connect(mac, address_type=pygatt.BLEAddressType.random)
+    finally:
+        adapter.stop()
+
+    return device
+
```

Comment:
Found a potential problem with pygatt. Let me investigate.

Reply:
Found and fixed the issue. Seems to be related to adapter.start() that tries to run a command that doesn't exist. 

This fixes the problem : adapter.start(reset_on_start=False)